CheckerUI_xforms/CheckerUI/CheckerUI/CheckerUI/Helpers/Order/OrderItemBuilder.cs

```diff
@@ -1,26 +1,31 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Text;
+using CheckerUI.Enums;
+using CheckerUI.ViewModels;
 
 namespace CheckerUI.Helpers.Order
 {
     internal static class OrderItemBuilder
```

Comment:
I think this is only relevant to mock data, since OrderItem's fields are all actually accessible properties, so when registering a new OrderItem, would it not be easier to just use
` OrderItem item = new OrderItem(){PropA =" ", PropB=" ", PropC=3,...};
`

Reply:
This way the changes if and when there are any item can be easier because we will initialize them there and not drag to the whole project and also in my opinion it will be replaced with components that will prepare the items read from the server